src/pgcopy/src/copy.rs

```diff
@@ -60,17 +61,28 @@ pub fn encode_copy_row_binary(
     Ok(())
 }
 
+/// Encodes given `row` into bytes with default param values
+/// in `CopyTextFormatParams`.
+pub fn encode_copy_row_text_default(
+    row: Row,
+    typ: &RelationType,
+    out: &mut Vec<u8>,
+) -> Result<(), io::Error> {
+    encode_copy_row_text(row, typ, out, CopyTextFormatParams::default())
+}
+
 pub fn encode_copy_row_text(
     row: Row,
     typ: &RelationType,
     out: &mut Vec<u8>,
+    CopyTextFormatParams { null, delimiter }: CopyTextFormatParams,
```

Comment:
API design nit: it's generally best to keep "config" parameters first (i.e., parameters that are unlikely to change if you're calling the method repeatedly in a loop on varying data), and out parameters last.

```suggestion
    CopyTextFormatParams { null, delimiter }: CopyTextFormatParams,
    row: Row,
    typ: &RelationType,
    out: &mut Vec<u8>,
```

I know the existing code isn't great about this, but I'll try to clean it up at some point in the future.

Reply:
Ack, these are all updated!